iterations.r

```diff
@@ -62,7 +65,9 @@ stations_metadata_df %>%
   transform_volumes() %>% 
   ggplot(aes(x=from, y=volume)) + 
   geom_line() + 
-  theme_classic()
+  theme_classic() +
+  #Task 6 - Add titles
+  labs(title = paste("Station: ", stations_metadata_df$id), x = "Date", y = "Volume")
```

Comment:
The plot looks good, but the title is not corresponding to the station. Now it picks the first row of stations_metadata_df and not the row in the sampled df. Here is how you could have done it:
```
# Sample one station that has data within the last 7 days
sampled_station <- stations_metadata_df %>% 
  filter(latestData > Sys.Date() - days(7)) %>% 
  sample_n(1)

# Use the ID from the sampled station to make the query
sampled_station %$% 
  vol_qry(
    id = id,
    from = to_iso8601(latestData, -4),
    to = to_iso8601(latestData, 0)
  ) %>% 
  GQL(., .url = configs$vegvesen_url) %>%
  transform_volumes() %>% 
  ggplot(aes(x=from, y=volume)) + 
  geom_line() + 
  theme_classic() +
  # Add titles using ID from the sampled station
  labs(title = paste("Station: ", sampled_station$id), x = "Date", y = "Volume")
```

Reply:
Also, I would consider to have the `name` in the plot title instead of the `id` variable.